<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralAssetForm</class>
 <widget class="QWidget" name="GeneralAssetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="weaponLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="amountSpinBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="acquireButton">
       <property name="text">
        <string>Acquire</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="currentLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pendingLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
